CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ All notable changes to [`@bpmn-io/properties-panel`](https://github.com/bpmn-io/
 
 ___Note:__ Yet to be released changes appear here._
 
+* `FEAT`: improve FEEL popup lifecycle events:
+  - `feelPopup.open` - fired before the popup is mounted
+  - `feelPopup.opened` - fired before the popup is unmounted. Event context contains the DOM node of the popup
```

Comment:
```suggestion
  - `feelPopup.opened` - fired after the popup is mounted. Event context contains the DOM node of the popup
```

Reply:
Or remove it from the changelog as @nikku suggested.

Reply:
Good catch. I'll remove it from the changelog and paste it into the description of this PR